tests/node_converters/lrn_test.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import onnx
+
+from tests.utils.common import check_onnx_model
+from tests.utils.common import make_model_from_nodes
+
+
+def _test_lrn(data: np.ndarray, alpha: float, beta: float, bias: float, size: int) -> None:
+    test_inputs = {'input_tensor': data}
+    node = onnx.helper.make_node(
+        op_type='LRN',
+        inputs=list(test_inputs),
+        outputs=['y'],
+        alpha=alpha,  # ONNX attributes are passed as regular keyword arguments.
+        beta=beta,
+        bias=bias,
+        size=size,
+    )
+
+    model = make_model_from_nodes(
+        nodes=node,
+        initializers={},
+        inputs_example=test_inputs,
+    )
+    check_onnx_model(model, test_inputs)
+
+
+def test_lrn() -> None:  # pylint: disable=missing-function-docstring
+    shape = (227, 227, 3)
+    data = np.random.random_sample(shape).astype(np.float32)
+    alpha = np.random.uniform(low=0.0, high=1.0).astype(np.float32)
+    beta = np.random.uniform(low=0.0, high=1.0).astype(np.float32)
+    bias = np.random.uniform(low=1.0, high=5.0).astype(np.float32)
+    size = np.random.randint(low=1, high=10)
```

Comment:
`size` should be odd, for example `n = size = 5`:

```
    center
      v
o x x c x x o o o o o (neurons)
  ---   ---
  n/2   n/2

o o x x c x x o o o o (next step)
o o o x x c x x o o o (next step)
o o o o x x c x x o o (next step)
...
```

To generate random odd you can use `randrange` from `random`:

```python
>>> import random
>>> random.randrange(start=1, stop=10, step=2)
9
>>> random.randrange(start=1, stop=10, step=2)
9
>>> random.randrange(start=1, stop=10, step=2)
1
>>> random.randrange(start=1, stop=10, step=2)
3
>>> random.randrange(start=1, stop=10, step=2)
1
>>> random.randrange(start=1, stop=10, step=2)
7
>>> random.randrange(start=1, stop=10, step=2)
9
>>> random.randrange(start=1, stop=10, step=2)
3
```

Reply:
Also, to verify your test you can to install test dependencies:

```
pip install -r test-requirements.txt
```
and run pytest for your test:
```
pytest tests/node_converters/lrn_test.py
```

Reply:
Ah, thank you for clarifying that size needs to be odd and suggesting `random.randrange`, I'll use that. I had thought size acted as a radius and not a diameter, oops!

I just tested the test as well, and it works! with a caveat--right now, I'm testing on an M2 MacBook Pro that doesn't have NVIDIA GPUs, so CUDA isn't available for my machine. As such, I modified `tests/utils/common.py` to skip the cpu vs. CUDA check in `check_onnx_model()` if `torch.cuda.is_available()` returns False. (Not sure if I wrote it the most pythonically) Alas, I can't confirm that that test will pass right now 😅

Reply:
This code will be more pythonic :)
```python
    _check_onnx_model(
        onnx_model=onnx_model,
        onnx_inputs=onnx_inputs,
        onnx_torch_check_function=onnx_torch_check_function,
        torch_cpu_cuda_check_function=torch_cpu_cuda_check_function if cuda_available else None,
        onnx_torch2onnx_check_function=onnx_torch2onnx_check_function,
        ignore_export_checker=ignore_export_checker,
        opset_version=opset_version,
    )
```

Could you revert "cuda_available" logic, because it skips all cuda-relative tests without any warnings/errors on CPU-only PC.
We will fix this problem later.

Reply:
ahhhh, I suspected there was a way to write the if-else inline, but I wasn't sure of the syntax!

I will go ahead and revert the change now (and just not run the tests anymore).